.gitignore

```diff
@@ -18,3 +18,5 @@
 
 .idea/libraries
 .idea/
+node_modules/
+package-lock.json
```

Comment:
The `package-lock.json` shoultn't be in the gitignore file.

Reply:
removed package-lock.json from gitignore and added file package-lock.json to repo